Apps/Sandcastle/gallery/Google Earth Enterprise.html

```diff
@@ -27,6 +27,8 @@
     function startup(Cesium) {
         'use strict';
 //Sandcastle_Begin
+// Add terrain and imagery data from a google Earth Enterprise server.
+// Imagery Layers tutorial: http://cesiumjs.org/tutorials/Imagery-Layers-Tutorial/
```

Comment:
Throughout, perhaps change these comments to something like "Also see the Imagery Layers..."

I also wonder if it is trivial to make this clickable hyperlinks (or are they already).  @emackey would know.